requirements.txt

```
@@ -24,7 +24,7 @@ pyparsing>=2.4
 python-dateutil>=2.8.1
 pytz>=2021.3
 requests>=2.20
-ruamel.yaml>=0.16
+ruamel.yaml>=0.18 # TODO: revert to 0.16 before merge assuming tests pass
```

```suggestion
ruamel.yaml>=0.16
```
Temporarily force installing `0.18` just to ensure no tests break.